pkg/k8shandler/common_test.go

```diff
@@ -46,15 +47,15 @@ var (
 
   6. common has request only
      node has no settings
-     -> request and limit set to be same and use common settings
+     -> request and mem limit set to be same and use common settings
```

Comment:
i think this would clearer if it said `mem request and limit set to be the same.`

Reply:
request in this case is both cpu and mem... i'll update to be more explicit